iot-e2e-tests/common/src/test/java/tests/integration/com/microsoft/azure/sdk/iot/iothub/setup/DeviceTwinCommon.java

```diff
@@ -431,6 +430,17 @@ public DeviceTwinTestInstance(IotHubClientProtocol protocol, AuthenticationType
             this.privateKey = privateKey;
             this.x509Thumbprint = x509Thumbprint;
             this.clientType = clientType;
+
+            try
+            {
+                this.sCDeviceTwin = DeviceTwin.createFromConnectionString(iotHubConnectionString);
+            }
+            catch (IOException e)
+            {
+                // A little odd to do this, but the createFromConnectionString call doesn't ever actually throw this exception, and
```

Comment:
so why are we doing it?

Reply:
I'll get rid of this and just have every twin test constructor declare that they throw this instead